staff/aleix-caminal/skylab-precurs/README.md

```diff
@@ -0,0 +1,31 @@
+![Logo Skylab](https://pbs.twimg.com/profile_images/718411051340079104/xJ-vRtnD_400x400.jpg)
```

Comment:
We had to do simple projects, it did not take almost 600 files. It is impossible to review them all ...

Reply:
its a node project, lots of files but you'll see that a lot

Reply:
changed it for a main anyways